test/actions/pulp3/repository/save_versions_test.rb

```diff
@@ -81,5 +81,24 @@ def test_save_new_version_from_lookup
       assert_equal task.output[:contents_changed], true
       assert_empty task.output[:updated_repositories] - [@repo1.id, @repo2.id]
     end
+
+    def test_save_version_with_outdated_publication
```

Comment:
Is this the correct file for the test?

I kinda doubt it, but it seemed to be the best fir without creating a new test-class  :innocent:

Reply:
Works for me!